nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -1142,8 +1155,8 @@ def test_forward_rel_ops():
     test_forward_pad()
     test_forward_gather()
     test_forward_stridedslice()
-    test_forward_split()
-    test_forward_unstack()
+    #test_forward_split()
+    #test_forward_unstack()
```

Comment:
Why are tests for `split` and `unstack` commented out?

Reply:
Thanks for pointing out. I have resubmitted the changes.